src/components/find-on-shelf/FindOnShelfModal.tsx

```diff
@@ -78,8 +78,13 @@ const FindOnShelfModal: FC<FindOnShelfModalProps> = ({
   });
 
   // Sorting of the data below to show branches & manifestations in the correct order.
+  const finalDataAlphabetical = finalData.sort(
+    (a: ManifestationHoldings, b: ManifestationHoldings) => {
+      return a[0].holding.branch.title > b[0].holding.branch.title ? 1 : -1;
```

Comment:
Consider using [`localeCompare()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) when comparing strings for sorting.

Reply:
Great suggestion, thx!
After some research and considerations, I opted in to follow it :) There are not that many branches we need to sort through so the [not-greatest-speed](https://stackoverflow.com/questions/14677060/400x-sorting-speedup-by-switching-a-localecompareb-to-ab-1ab10) of .localeCompare() won't burden us.